README.md

```diff
@@ -1293,13 +1292,18 @@ import * as yup from 'yup';
 
 const personSchema = yup.object({
   firstName: yup
-    .string(),
+    .string()
+    .defined(),
```

Comment:
I think it's worth noting why `defined()` is being used here since it's not "idiomatic yup" (to the extent that's a thing)